system/ee/EllisLab/ExpressionEngine/Library/Filesystem/Filesystem.php

```diff
@@ -538,7 +538,7 @@ public function chmod($path, $mode)
 	public function isWritable($path)
 	{
 		// If we're on a Unix server with safe_mode off we call is_writable
-		if (DIRECTORY_SEPARATOR == '/' AND @ini_get("safe_mode") == FALSE)
```

Comment:
Based on the comments in this code we should leave `safe_mode` check where it is